inc/html.class.php

```diff
@@ -6956,6 +6976,43 @@ public static function compileScss($args) {
       return $css;
    }
 
+   /**
+    * Returns SCSS file hash.
+    * This function evaluates recursivly imports to compute a hash that represent the whole
+    * contents of the final SCSS.
+    *
+    * @param string $filename
+    *
+    * @return null|string
+    */
+   public static function getScssFileHash(string $filename) {
```

Comment:
Should be unit tested

Reply:
I've put this in a separated PR with unit tests: https://github.com/glpi-project/glpi/pull/6482